### PR TITLE
add unit tests for the monitor api and credential manage api

### PR DESCRIPTION
**Pull request by wanglin100**

### CODE REVIEW

File: runtime/src/test/java/org/eclipse/xpanse/runtime/CredentialManageApiTest.java [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+                        List.of(new CredentialVariable("name", "description"))));
+        when(mockCredentialCenter.getCredentialDefinitionsByServiceId("id",
+                CredentialType.VARIABLES)).thenReturn(credentialDefinitions);
+
+        // Run the test
+        final List<CredentialDefinition> result =
+                credentialManageApiUnderTest.getCredentialDefinitionsByServiceId("id",
+                        CredentialType.VARIABLES);
+
+        // Verify the results
```
Comment (swaroopar) [inline runtime/src/test/java/org/eclipse/xpanse/runtime/CredentialManageApiTest.java:296]: assertions missing?

**Review by swaroopar — CHANGES_REQUESTED** [review]
comment added

File: runtime/src/test/java/org/eclipse/xpanse/runtime/CredentialManageApiTest.java (hunk 1 above)
Comment (wanglin100) [reply runtime/src/test/java/org/eclipse/xpanse/runtime/CredentialManageApiTest.java:299]: I have modified

**Review by swaroopar — APPROVED** [review]
looks good.

**Pull request merged**